packages/cardpay-subgraph/src/mappings/gnosis-safe.ts

```diff
@@ -122,7 +122,7 @@ export function handleExecutionSuccess(event: ExecutionSuccess): void {
         safeTxEntity.safeTxGas.toString(),
         safeTxEntity.baseGas.toString(),
         safeTxEntity.gasPrice.toString(),
-        safeTxEntity.gasToken,
+        safeTxEntity.gasToken as string,
```

Comment:
Addressing `ERROR TS2322: Type '~lib/string/String | null' is not assignable to type '~lib/string/String'`